components/pkg/utils/snowflake.go

```diff
@@ -0,0 +1,379 @@
+//
+// Copyright 2021 Layotto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package utils
```

Comment:
Shall we move these two files from `utils` package to `sequencer/snowflake` package?
The code under `utils` package should be reusable, but these two files can't be used by other components
![image](https://user-images.githubusercontent.com/26001097/196845406-61e440cf-6f23-4178-828a-c508acfb8b33.png)

Reply:
got it